ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/ServerClusterId.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonSubTypes({})
+public class ServerClusterId {
+  private static final String UNUSED_ID = "";
+  private static final String KAFKA_CLUSTER = "kafka-cluster";
+  private static final String KSQL_CLUSTER = "ksql-cluster";
+
+  @JsonProperty("id")
+  private final String id;
+
+  @JsonProperty("scope")
+  private final Map<String, String> scope;
+
+  @JsonCreator
+  ServerClusterId(
+      @JsonProperty("id") final String id,
```

Comment:
I separated `of` and the constructor so that I could accept only 2 clusters in the `of`, but have a `scope` JSON property name. I don't want to give the freedom to accept any map in the `of`. How do I wrap 2 parameters into one `scope` property?